src/main/java/mate.academy/Main.java

```diff
@@ -8,9 +8,8 @@
 public class Main {
 
     public static void main(String[] args) {
-        // Please test your Injector here. Feel free to push this class as a part of your solution
         Injector injector = Injector.getInjector();
-        ProductService productService = null;
+        ProductService productService = (ProductService) injector.getInstance(ProductService.class);
```

Comment:
Casting the result of `getInstance()` to `ProductService` is necessary here, but make sure that the Injector is responsible for checking if the correct type is returned. If the wrong type is injected, there should be a clear and informative exception message.